server/index.js

```diff
@@ -77,8 +77,8 @@ const proxy = FAKE_V1_API
           : "http://"
       }${PROXY_HOSTNAME}`,
       changeOrigin: true,
-      proxyTimeout: 3000,
-      timeout: 3000,
+      proxyTimeout: 10000,
+      timeout: 10000,
```

Comment:
What I did was I injected a `import time; time.sleep(5)` into `kuma/api/v1/views.py` inside the `whoami` function. Just to make it easier to simulate a slow XHR request.